### PR TITLE
Apply /contact-us redesign

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Applied redesign as per [doc](https://docs.google.com/document/d/1KkLHMrGipY4zgIXreujpyJlLvYUs-iAlP-FtpYrZmWA/edit) and [design](https://www.figma.com/file/0DTIS3uNNUuDAdCKWrk8Uf/23.10---Microk8s.io?node-id=407%3A6063&mode=dev)

## QA

- View the site locally in your web browser at: https://microk8s-io-611.demos.haus/contact-us
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4196

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
looks great, thanks @mtruj013

**Review by anthonydillon — CHANGES_REQUESTED** [review]
The form in Marketo does not match the form here: 
image

File: static/sass/styles.scss [hunk 1]
```diff
@@ -117,3 +117,9 @@
     overflow: scroll;
   }
 }
+
+.p-list--divided.is-contact {
+  li {
+    padding-bottom: 0;
+  }
+}
```
Comment (anthonydillon) [inline static/sass/styles.scss:125]: Personally, I would add `u-no-padding--bottom` to the li's instead of a new class.

**Review by anthonydillon — CHANGES_REQUESTED** [review]

**Comment by juanruitina** [comment]
* On mobile, the consent checkbox and the label are not aligned.
* Can you please add a bit of space between that and the "By submitting this form..." line, to make clear they are not related?

Screenshot 2023-08-18 at 16 38 37

LGTM otherwise!

**Comment by mtruj013** [comment]
Thanks all! @juanruitina and @anthonydillon could you take another look? Sophie solved the marketo fields issue, we're using a new form so the id has been updated

**Review by anthonydillon — CHANGES_REQUESTED** [review]
The form looks good but First name, Last name, Email and Phone number are required fields in Marketo.

Could you use the [Vanilla pattern for required fields](https://vanillaframework.io/docs/base/forms#required)?

**Comment by mtruj013** [comment]
Thanks @anthonydillon, could you take another look? I updated the required fields to use the Vanilla pattern, but omitted the legend as this is already accounted for in the copy (`"* Mandatory field"` further down in the page)

**Review by anthonydillon — APPROVED** [review]
Nice, thanks @mtruj013

**Comment by juanruitina** [comment]
LGTM!

**Pull request merged**